### DISCUSSION TITLE
是否支持将笔记数据文件映射到宿主机

### DISCUSSION THREAD

**Discussion by suppergit** [usememos/memos#39] [Ideas]
目前应用只支持数据导出，不支持导入，能否将笔记数据文件映射到宿主机，这样更新docker的时候也不会丢失数据

**Comment by boojack** [comment]
You can mount a `.db` file to a directory when running a docker image. 
`docker run --name memos --publish 8080:8080 --volume ~/path/to/your/data/:/var/opt/memos -e mode=release -e data=/var/opt/memos neosmemo/memos:next`
